apps/communications/contacts/js/contacts.js

```diff
@@ -1082,6 +1112,12 @@ var Contacts = (function() {
     },
     get SHARED_CONTACTS() {
       return SHARED_CONTACTS;
+    },
+    get isSelectorReady() {
```

Comment:
I think this is not the proper way of implementing this and not really sure if this is necessary at all